test/ValidatorSetAuRa.js

```diff
@@ -11,6 +11,13 @@ require('chai')
   .use(require('chai-bn')(BN))
   .should();
 
+const initialValidators = [
```

Comment:
The integration test repo solves this by committing the secret keys to the git repo.  That is not a vulnerability, since the keys are only used for tests and hold no assets.

Reply:
Anyway, we can't use hard-coded addresses in unit tests. Try to write some simple unit test and send some transaction from the hard-coded address in it. You will get an error because `truffle` and `ganache-cli` don't know the private key of that address.